config/stack_noaa.yaml

```diff
@@ -53,25 +53,25 @@ netcdf:
   version_cxx: 4.3.1
 
 nccmp:
-  build: YES
+  build: NO
   version: 1.8.7.0
 
 nco:
   build: NO
   version: 4.9.3
 
 cdo:
-  build: YES
+  build: NO
```

Comment:
please revert to develop

Reply:
I changed the build value for cdo to YES, although I did not test with this on.  I will retest once changes are complete.